containers/ecr-viewer/playwright.config.ts

```diff
@@ -43,9 +43,10 @@ export default defineConfig({
     },
   ],
   webServer: {
-    command: "npm run local-dev",
+    command: "npm run local-build-start",
```

Comment:
why switch to the built vs dev? could we have used `local-docker` instead of adding these new scripts?